bridge/opentracing/bridge.go

```diff
@@ -503,19 +520,19 @@ func otTagToOTelLabel(k string, v interface{}) label.KeyValue {
 	case int64:
 		return key.Int64(val)
 	case uint64:
-		return key.Uint64(val)
+		return key.String(fmt.Sprintf("%d", val))
 	case float64:
 		return key.Float64(val)
 	case int32:
-		return key.Int32(val)
+		return key.Int64(int64(val))
 	case uint32:
-		return key.Uint32(val)
+		return key.Int64(int64(val))
 	case float32:
-		return key.Float32(val)
+		return key.Float64(float64(val))
 	case int:
 		return key.Int(val)
 	case uint:
-		return key.Uint(val)
+		return key.Int(int(val))
```

Comment:
Similar to the `uint64`->`int64` conversion, `uint`->`int` is not safe as they are both the same size.  This should probably also convert to `string` to avoid overflow issues.

```suggestion
		return key.String(fmt.Sprintf("%d", val))
```